src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -1,8 +1,8 @@
-An *Infrastructure Manager* is responsible for deploying nodes. In most of the cases it knows how to launch a node
+A *Node Source Infrastructure* also called *Infrastructure Manager* is responsible for deploying ProActive Nodes inside a defined <<_deploy_nodes_from_proactive_rm,*Node Source*>>. In most of the cases, it knows how to launch a set of ProActive Nodes
```

Comment:
A *Node Source Infrastructure*, also called *Infrastructure Manager*, is responsible....  [missing commas]